frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -39,7 +39,8 @@ const podStatusIsNumeric = (podStatusValue: string) => {
   return (
     podStatusValue !== 'Scaled to 0' &&
     podStatusValue !== 'Autoscaled to 0' &&
-    podStatusValue !== 'Idle'
+    podStatusValue !== 'Idle' &&
+    podStatusValue !== 'Scaling Up'
```

Comment:
Since these values are used in multiple places, it would be good to have enum defined for posStatusValue.

Reply:
add a enum `AllPodStatus`.